.github/workflows/build_and_upload_docker_image_on_push.yaml

```diff
@@ -0,0 +1,88 @@
+name: Build & Upload Fairspace-Jupyter Docker image
+
+on:
+  push:
+    branches:
+      - dev
+      - release
+
+jobs:
+  build-and-upload-jupyterhub:
+    name: Build & Upload Fairspace-Jupyter (K8s Hub) Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Get version tag for artifact
+        run: |
+          BRANCH=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}
+          echo "Building images from the branch: $BRANCH"
+          VER=$(cat ./projects/jupyterhub-hub/VERSION)
```

Comment:
it looks like we've been using the main project ./VERSION file for image tags, I see that the project specific ones haven't been updated for 5 years... 
I think it makes sense to have single version file for this project (and to remove project specific ones), otherwise it will be easy to create inconsistencies and confusion which hub image works with which singleuser one, keep it also consistent with helm chart version.

Reply:
agree, done